TestingHomework-Discounts/Managers/CheckoutManager.cs

```diff
@@ -2,206 +2,47 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using TestingHomework_Discounts.Accessors;
 
 namespace TestingHomework_Discounts.Managers
 {
-    public class CheckoutManager
```

Comment:
The managers have become needless baggage since all of the logic has been moved to the accessor. This undermines both our architecture/separation of concerns and testing. Since all the logic is in the accessor, we can't mock out things like database access to test our logic in isolation.

We also don't gain the benefit of simplified tests through smaller & fewer concerns to test at a time